dev/tests/api-functional/testsuite/Magento/AggregatedServices/Api/Service/GetGuestCartTest.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AggregatedServices\Api;
+
+class GetGuestCartTest extends \Magento\TestFramework\TestCase\WebapiAbstract
+{
+    const SERVICE_NAME = 'aggregatedServicesGuestAggregatedCarts';
```

Comment:
I don't think it's correct, will double check.

Reply:
The test on CI will fail if it's incorrect.